src/app/App.tsx

```diff
@@ -1,19 +1,31 @@
-import { AppLocale } from 'context/locale/appLocale.enum';
-import { useLocale } from 'hooks';
+import { useState } from 'react';
+import { useCheckMobileScreen } from 'hooks';
+import * as Styled from './App.styles';
+import { Main } from './components/main/Main';
+import { Sidebar } from './components/sidebar/Sidebar';
 
 export const App = () => {
-  const { formatMessage, locale, setLocale } = useLocale();
+  const [mobileFiltersVisable, setMobileFiltersVisable] = useState<boolean>(false);
```

Comment:
To byłoby dobrze wrzucić do komponentu `Sidebar`, szczególnie że jego wersja mobilna też się powinna tam znaleźć.
Druga sprawa - jest taka konwencja nazewnictwa, że wszelkie booleany zaczynamy prefixem `is...`. Więc tutaj będzie `isMobileFiltersVisible` a poniżej `isScreenMobile`